app/containers/FileModal.js

```diff
@@ -40,6 +47,19 @@ const styles = StyleSheet.create({
 		flex: 1
 	}
 });
+const handleSave = (img) => {
+	FileSystem.downloadAsync(img, `${ FileSystem.documentDirectory + img[0] + Math.floor(Math.random() * 1000) }.jpg`)
+		.then(({ uri }) => {
+			CameraRoll.saveToCameraRoll(uri).then(() => {
+				EventEmitter.emit(LISTENER, { message: I18n.t('saved_to_gallery') });
+			}).catch(() => {
+				EventEmitter.emit(LISTENER, { message: I18n.t('error-save-image') });
+			});
+		})
+		.catch(() => {
+			EventEmitter.emit(LISTENER, { message: I18n.t('error-save-image') });
+		});
+};
```

Comment:
I think this can be better if you use `async/await` instead `then/catch`.
```
try {
  const file = `${ FileSystem.documentDirectory + img[0] + Math.floor(Math.random() * 1000) }.jpg`;
  const { uri }  = await FileSystem.downloadAsync(img, file);
  await CameraRoll.saveToCameraRoll(uri);
} catch (e) {
  EventEmmiter.emit(LISTENER, { message: I18n.t('error-save-image') });
}
```

Reply:
@djorkaeffalexandre Definitely async/await is better, it made the code so concise without having to write the same catch block twice. Thanks for this review Sir.